.htaccess

```diff
@@ -35,9 +35,9 @@ FileETag MTime Size
 	RewriteRule src/Backend/Cache/Locale/.*\.php - [F]
 
 	# redirect all trafic to https
-	# RewriteCond %{SERVER_PORT} 80
+	# RewriteCond %{HTTPS} != on
 	# RewriteCond %{HTTP_HOST} !.*\.dev [NC]
-	# RewriteRule ^(.*)$ https://www.<domain>.be/$1 [R=301,L]
+	# RewriteRule ^ https://%{HTTP_HOST}%{REQUEST_URI} [R=301,L]
```

Comment:
@jessedobbelaere is that something we can enable by default or should it only be enabled with https